Tests/NetworkingTests/URLRequest+ExtensionsTests.swift

```diff
@@ -2,37 +2,9 @@
 import XCTest
 
 final class URLRequestTests: XCTestCase {
-    var sut: URLRequest!
-    
-    override func setUp() {
-        super.setUp()
-        sut = URLRequest(url: URL(string: "https://www.google.com")!)
-    }
-    
-    override func tearDown() {
-        sut = nil
-        super.tearDown()
-    }
-}
-
-extension URLRequestTests {
-    func test_tokenExchange() throws {
-        let tokenRequest = sut.tokenExchange(subjectToken: "tesSubjectToken", scope: "testScope")
-        let contentTypeHeader = tokenRequest.value(forHTTPHeaderField: "Content-Type")
-        let httpMethod = tokenRequest.httpMethod
-        let httpBody = try XCTUnwrap(tokenRequest.httpBody)
-        let body = String(decoding: httpBody, as: UTF8.self).split(separator: "&")
-        XCTAssertEqual(tokenRequest.url, URL(string: "https://www.google.com"))
-        XCTAssertEqual(contentTypeHeader, "application/x-www-form-urlencoded")
-        XCTAssertEqual(httpMethod, "POST")
-        XCTAssertEqual(body[0], "grant_type=urn:ietf:params:oauth:grant-type:token-exchange")
-        XCTAssertEqual(body[1], "scope=testScope")
-        XCTAssertEqual(body[2], "subject_token=tesSubjectToken")
-        XCTAssertEqual(body[3], "subject_token_type=urn:ietf:params:oauth:token-type:access_token")
-    }
-    
     func test_authorized() throws {
-        let authorizedRequest = sut.authorized(with: "testBearerToken")
+        let url = try XCTUnwrap(URL(string: "https://www.google.com"))
+        let authorizedRequest = URLRequest(url: url).authorized(with: "testBearerToken")
```

Comment:
Worth just using your `.example` URLRequest and removing a line of code?

Reply:
yep, nice!

Reply:
Done ✅